src/components/home/HomePage.scss

```diff
@@ -0,0 +1,351 @@
+#navbar {
+  display: flex;
+  height: 100px;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #6676a0;
+  color: white;
+  justify-content: flex-end;
+  position: relative;
+  padding: 0;
+  overflow-x: hidden;
+
+  .collapseMenuItems {
+    width: 80%;
+    justify-content: flex-end;
+  }
+
+  .navbarItems {
+    width: 50%;
+    padding: 10px 5px 10px 5px;
+
+    > .col {
+      padding-left: 5px;
+      padding-right: 5px;
+      transition: font-size 0.3s ease-in-out;
+    }
+  }
+
+  .navbarLogo {
+    position: absolute;
+    max-width: 100px;
+    left: 100px;
+    cursor: pointer;
+  }
+
+  > div {
+    text-align: center;
+
+    a {
+      text-decoration: none;
+      color: white;
+    }
+  }
+
+  .iconTrio {
+    width: 25%;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: 20px;
+    padding: 10px 5px 10px 5px;
+
+    > div {
+      max-width: 35px;
+      margin-right: 15px;
+      min-width: 18px;
+    }
+
+    .fas,
```

Comment:
im guessing these are fontawesome. can you add a comment above these lines and anywhere else .fas and .fab are being used that its fontawesome, no one can guess it just from the words fas and fab